xdsl/printer.py

```diff
@@ -665,9 +665,11 @@ def print_op_with_default_format(self, op: Operation) -> None:
 
         self.print_regions(op.regions)
         self.print_op_attributes(op.attributes)
+        self.print(" : ")
+        self.print_operand_type(op)
 
-        # Print the operation type
-        self.print(" : (")
+    def print_operand_type(self, op: Operation) -> None:
```

Comment:
Can you rename this to `print_operation_type`?

Reply:
Renamed in new commit.